sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -28,4 +28,32 @@ export class ApplicationSystemController {
 
     return { data: application.data };
   }
+
+  @Patch(":applicationId/assessment")
+  async updateAssessmentInApplication(
+    @Body() assessment: ApplicationAssessmentDTO,
+    @Param("applicationId") applicationId: number,
+  ): Promise<number> {
+    const application = await this.getByApplicationId(applicationId);
```

Comment:
I would recommend do not retrieve the entire record to just update on a particular column.
Since we have the application id and the value to be updated, we can just update the column, IMO.